### ISSUE TITLE
Allow commands to be executed in a specific order with intervals

### ISSUE THREAD

**Issue by wnjdxsj** [ImCodist/quick-menu#18]
Allow commands to be executed in a specific order with intervals.
For example:
A says: 1 -1s 2 -1s 3
Instead of:
A says: 1 2 3 (No intervals)

As a temporary solution for those who want this functionality:
Maybe you can place some command blocks, and use this mod to trigger them (fill a redstone block or summon PvP Boss?).

**Comment by zz32k** [comment]
Yeah this would be great, since I can't use multiple commands in one button on my server since I get kicked from running commands too fast
